src/withPropsStream.js

```diff
@@ -0,0 +1,16 @@
+import {withLatestFrom} from 'rxjs/operator/withLatestFrom';
+import wrap from './utils/wrap';
+import invariant from './utils/invariant';
+import mapPropsStream from './mapPropsStream';
+import createHelper from './createHelper';
+
+const withPropsStream = propsStreamMapper => mapPropsStream((props$, obs) => {
+  const nextProps$ = wrap(propsStreamMapper)(props$, obs);
+  invariant(typeof nextProps$.subscribe === 'function', 'Expected a props Observable.');
```

Comment:
It is.

Still, I should probably have introduced this `invariant` helper in another PR... bear with me :)